src/httprpc.cpp

```diff
@@ -93,10 +93,8 @@ static bool multiUserAuthorized(std::string strUserPass)
     std::string strUser = strUserPass.substr(0, strUserPass.find(":"));
     std::string strPass = strUserPass.substr(strUserPass.find(":") + 1);
 
-    if (gArgs.IsArgSet("-rpcauth")) {
-        //Search for multi-user login/pass "rpcauth" from config
-        BOOST_FOREACH(std::string strRPCAuth, gArgs.GetArgs("-rpcauth"))
-        {
+    //Search for multi-user login/pass "rpcauth" from config
+    BOOST_FOREACH(std::string strRPCAuth, gArgs.GetArgs("-rpcauth")) {
```

Comment:
We generally prefer to remove BOOST while changing lines, no (also, I believe could constify the strRPCAuth, while you're at it)?